server/controllers/socialLoginController.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable object-curly-newline */
+import Model from '../models';
+import Authenticator from '../middlewares/authenticator';
+
+
+const { User } = Model;
+const { generateToken } = Authenticator;
+
+/**
+ * @class SocialLoginController
+ */
+class SocialLoginController {
+  /**
+   * @description - find  or create a new user and pass the user to the request body
+   * @param {object} user
+   * @param {function} done
+   * @returns {object} createOrFindUser
+   */
+  static modelQuery(user, done) {
```

Comment:
@udochukwu kindly use a more meaningful name for this method, a name that describes what it is doing.